diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -54,7 +54,7 @@ SUPPORTED_ARCHS = {"8.0", "8.6", "8.7", "8.9", "9.0"}
 # Compiler flags.
-CXX_FLAGS = ["-g", "-O3", "-fopenmp", "-lgomp", "-std=c++17", "-DENABLE_BF16"]
+CXX_FLAGS = ["-g", "-O3", "-fopenmp", "-lgomp", "-std=c++20", "-DENABLE_BF16"]
 NVCC_FLAGS = [
     "-O3",
-    "-std=c++17",
+    "-std=c++20",
     "-U__CUDA_NO_HALF_OPERATORS__",
     "-U__CUDA_NO_HALF_CONVERSIONS__",
